bsp/wch/risc-v/yd-ch32v307vct6/applications/main.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2024, GSunwinder
```

Comment:
Hi please use this comment
```suggestion
 * Copyright (c) 2006-2022, RT-Thread Development Team
```

Reply:
thanks for the tip